src/handlebar-helpers.js

```diff
@@ -36,6 +36,12 @@ const helpers = {
     } else {
       return -1;
     }
+  },
+  add: function(x, y) {
+    if ( typeof x === 'number' && typeof y === 'number') {
+      return x + y;
+    }
+    return 0;
```

Comment:
since we went with -1 for divide, i feel we should probably be consistent here as well
```suggestion
    return -1;
```